### ISSUE TITLE
Documentation on Postgres jsonb serialization is not consistent

### ISSUE THREAD

**Issue by nsimmons** [RailsEventStore/rails_event_store#1681]
My team recently changed our RES config to use PG jsonb serialization. We followed the docs here: https://railseventstore.org/docs/v2/mapping_serialization/#configuring-for-postgres-json-b-data-type

Only later on did we discover JSONClient that is mentioned at the bottom of this page: https://railseventstore.org/docs/v2/install/

The mapping serialization page should probably mention the JSONClient that solves the issue in blue text at the bottom of that page.

**Comment by pawelpacana** [comment]
Good catch! JSONClient should definitely be mentioned in the mapping serialization page. Can you please make the necessary adjustment? I'll happily accept the PR.
